app/assets/javascripts/activeadmin_addons/addons/interactive_select_tag.js

```diff
@@ -68,6 +68,7 @@ var initializer = function() {
         url: url,
         data: data,
         dataType: 'json',
+        headers : {'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')},
```

Comment:
Extra space after key 'headers'  key-spacing<br>A space is required after '{'    object-curly-spacing<br>A space is required before '}'   object-curly-spacing